tests/ci/stress.py

```diff
@@ -75,6 +75,9 @@ def get_options(i: int, upgrade_check: bool) -> str:
     if not upgrade_check:
         client_options.append("ignore_drop_queries_probability=0.5")
 
+    if random.random() < 0.2:
```

Comment:
Do we we also need to set other settings too?
```
max_parallel_replicas=3,
cluster_for_parallel_replicas='parallel_replicas',
parallel_replicas_for_non_replicated_merge_tree=1,
```

Reply:
I think yes